sim/src/image.rs

```diff
@@ -221,11 +221,11 @@ impl ImagesBuilder {
                 Box::new(BoringDep::new(image_num, deps))
             };
             let primaries = install_image(&mut flash, &slots[0],
-                maximal(42784), &ram, &*dep, false);
+                maximal(42784), &ram, &*dep, false, Some(0));
```

Comment:
Does it need to be "something" or could it be skipped (or an explicit None is required anyway)?

Reply:
If None is being used it will not include the TLV itself. Some(0) adds one with 0 as the initiall value

Reply:
The important thing is that the SECCNT TLV won't be added to the image if this capability is disabled,
regardless of the set value. Thanks!